charts/brigade/values.yaml

```diff
@@ -50,6 +50,9 @@ controller:
   #  requests:
   #    cpu:
   #    memory:
+  serviceAccount:
+    create: true
+    name: 
```

Comment:
I wonder if it would be clearer to comment this line out, perhaps with an associated comment mentioning it only needs to be set if a non-default name is desired?

Reply:
That sounds reasonable.

Reply:
I am noting this is aligned with the convention in other charts around `serviceAccount.name`, so I'm good with keeping as is -- defer to the author.